### ISSUE TITLE
[FEATURE] Add to docs how CPU over provisioning / CPU pinning works for guest VMs

### ISSUE THREAD

**Issue by wjimenez5271** [harvester/harvester#2082]
We should add to our docs the behavior of how a core allocated to a guest VM maps to physical cores on the host, including patterns of over-provisioning, how VM guests are scheduled shares of CPU time, etc.

**Comment by weihanglo** [comment]
Just FYI. We already have a post in our knowledge base: https://harvesterhci.io/kb/vm-scheduling. It is still better having a dedicated user manual for that tho.

**Comment by irishgordo** [comment]
Hi @weihanglo hope the week has been going well for you so far!

I would like to close this after this PR gets merged in:
https://github.com/harvester/docs/pull/151
In that PR above I made a small change to the spelling in the heading of the document.

After that PR gets merged in I believe we can close the document update, since that PR I created fixes the only issue I saw and since your repository fork's branch has already been merged into 'main'.
